src/main/java/com/booleanuk/core/Basket.java

```diff
@@ -1,5 +1,72 @@
 package com.booleanuk.core;
 
+import java.util.ArrayList;
+
 public class Basket {
 
+
+    ArrayList<String> list;
+    int capacity;
+
+    public Basket (){
+        this.list = new ArrayList<String>();
+        this.capacity = 3;
+    }
+
+    public String add(String bagel){
+        if(list.size() < capacity){
+            list.add(bagel);
+        }
+
+        return list.toString();
+    }
+
+    public String remove(String bagel){
+        if(list.contains(bagel)){
+            list.remove(bagel);
+        }else return "Bagel does not exist in basket";
+
+        return list.toString();
+    }
+
+    public String changeCapacity(int size) {
```

Comment:
I've marked this as a 3 but bear in mind that since we can always resize an ArrayList there is no need to copy the previous list into the new one when we resize, all we need to do is change the value of the capacity variable and keep track of it.